src/wasm/wasm-binary.cpp

```diff
@@ -2903,7 +2912,12 @@ void WasmBinaryReader::readSourceMapHeader() {
     uint32_t lineNumber =
       readBase64VLQ(*sourceMap) + 1; // adjust zero-based line number
     uint32_t columnNumber = readBase64VLQ(*sourceMap);
-    nextDebugLocation = {fileIndex, lineNumber, columnNumber};
+    std::optional<BinaryLocation> nameIndex = std::nullopt;
```

Comment:
```suggestion
    std::optional<BinaryLocation> nameIndex;
```
The default value is nullopt anyhow.